### DISCUSSION TITLE
Integrating Confluent kafka with racecar

### DISCUSSION THREAD

**Discussion by srikanthreddy919** [zendesk/racecar#253] [Q&A]
I have tried to implement Confluent Kafka with racecar but I am getting messages in Different format
#<Racecar::Message:0x00007ffa15b08be8 @rdkafka_message=#<Rdkafka::Consumer::Message:0x00007ffa15b09ea8 @topic="dv1.aws-use1.align.fct.doctor.v1", @partition=5, @payload="{\"count\":0}", @key="alice", @offset=85, @timestamp=2021-07-09 16:26:18 +0530, @headers={}>, @retries_count=0>

Please suggest any idea and solution

**Comment by dasch** [comment]
It looks like you are receiving messages just fine. Call e.g. `message.value` and `message.key` to retrieve the various parts of the message.

Also, read [this section of the README](https://github.com/zendesk/racecar#creating-consumers).
